firefox_puppeteer/tests/test_utils.py

```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness import FirefoxTestCase
```

Comment:
This won't work anymore. The testcase class is no longer part of the harness but puppeteer.  See all the other tests. The reason why Travis did not fail is that you haven't added the new file to the manifest. So best practice is always to check if your working branch has all the latest changes from the master/mozilla-central branch. 

Let me do the changes and push it, given that it somewhat blocks my work on bug 1232967.